src/composables/useBadgeData.js

```diff
@@ -178,13 +178,32 @@ export default function useBadgeData() {
 	 */
 	const badgeData = computed(() => combineBadgeData(badgeAchievementData.value, badgeContentfulData.value));
 
+	/**
+	 *
```

Comment:
For consistency with the other method jsdoc comments, add a general comment to the top and remove the `{*}`. Take a look at the other methods here for examples.